### PR TITLE
Set image name to id if name is nil in Glance

### PR DESCRIPTION
**Pull request by tzumainn**
https://bugzilla.redhat.com/show_bug.cgi?id=1438100

### CODE REVIEW

**Comment by Fryguy** [comment]
Please rebase...I believe master is now green.

File: app/models/manageiq/providers/openstack/refresh_parser_common/images.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@ def parse_image(image)
           :type               => self.class.miq_template_type,
           :uid_ems            => uid,
           :ems_ref            => uid,
-          :name               => image.name,
+          :name               => image.name.nil? ? image.id : image.name,
```
Comment (chessbyte) [inline app/models/manageiq/providers/openstack/refresh_parser_common/images.rb]: I think `image.name || image.id` is more Ruby-esque.

Comment (tzumainn) [inline app/models/manageiq/providers/openstack/refresh_parser_common/images.rb]: Ah, thanks for passing that along!  I'll make the change.

**Comment by simaishi** [comment]
Fine backport (to manageiq repo) details:

```
$ git log -1
commit 7098d6a6ac287fd2824cb0b830d69ad7f52c0ce3
Author: Jason Frey <fryguy9@gmail.com>
Date:   Mon Apr 10 10:58:53 2017 -0400

    Merge pull request #3 from tzumainn/image-with-no-name-refresh
    
    Set image name to id if name is nil in Glance
    (cherry picked from commit afd9f457eabbd65d0c8ac61e5037425853f777d4)
```

**Comment by simaishi** [comment]
Euwe backport (to manageiq repo) details:

```
$ git log -1
commit f73ac9d5b04e89e4da6d384a56dc3d4da91b736b
Author: Jason Frey <fryguy9@gmail.com>
Date:   Mon Apr 10 10:58:53 2017 -0400

    Merge pull request #3 from tzumainn/image-with-no-name-refresh
    
    Set image name to id if name is nil in Glance
    (cherry picked from commit afd9f457eabbd65d0c8ac61e5037425853f777d4)
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1442164
```

**Pull request merged**